lib/lunchbot/accounts.ex

```diff
@@ -189,10 +189,16 @@ defmodule Lunchbot.Accounts do
 
   """
   def update_user_password(user, password, attrs) do
+    # Our password will only be nil when setting the password after a Google login
     changeset =
-      user
-      |> User.password_changeset(attrs)
-      |> User.validate_current_password(password)
+      if(password == "") do
+        user
+        |> User.password_changeset(attrs)
```

Comment:
If a user sets a password, would logging in from google unset the password?

Reply:
Nope! When we call `fetch_or_create_user` we first check if the user is already in the database, and if the user is there we grab all of the info. We only have a nil password when we have a new user inserted into the database on a Google login